packages/cli-upload/src/upload.js

```diff
@@ -4,6 +4,14 @@ import command from '@percy/cli-command';
 import * as UploadConfig from './config.js';
 
 const ALLOWED_FILE_TYPES = /\.(png|jpg|jpeg)$/i;
+const ALLOWED_TOKEN_TYPES = ['web', 'generic'];
+
+// All BYOS screenshots have a fixed comparison tag
+export const BYOS_TAG = {
+  name: 'BYOS',
```

Comment:
```suggestion
  name: 'BYOS',
```

lets discuss name field here.